junction/templates/proposals/list.html

```diff
@@ -91,7 +91,7 @@
         </div>
 
         {% if proposals_to_review %}
-        {% include 'proposals/partials/proposal-list--items.html' with proposals=proposals_to_review title="Proposals To Be Reviewed" display_status=False %}
+        {% include 'proposals/partials/proposal-list--items.html' with proposals=proposals_to_review title="Proposals To Be Reviewed" display_status=False content_status=True %}
```

Comment:
This is always `content_status` is always going to be `true`, wondering why would you like to do something like this. You need implement actual logic for detecting content uploaded or not.